model_analyzer/perf_analyzer/perf_config.py

```diff
@@ -151,6 +151,21 @@ def representation(self):
         return PerfAnalyzerConfig.remove_url_from_cli_string(
             self.to_cli_string())
 
+    def extract_model_specific_parameters(self):
+        """
+        Returns a dictionary of parameters (options+args) that can change
+        between models for a given RunConfig
```

Comment:
Should this say ModelRunConfig? If not, a single dictionary won't suffice. It would need to be per-model dict.

Reply:
TMA-571 was filed to track this